morgoth/settings.py

```diff
@@ -166,6 +166,7 @@ class Development(Base):
     AUTH_PASSWORD_VALIDATORS = values.ListValue([])
     SECURE_SSL_REDIRECT = values.Value(False)
     SLEEP_SECONDS = values.IntegerValue(0)
+    BASIC_AUTH_ENABLED = values.Value(True)
```

Comment:
Instead of defining this only in the dev instance, I think you should define it in the Base configs, defaulting to False, and override it here. Then you don't have to do the `getattr` dance in `morgoth/base/decorators.py`.